### ISSUE TITLE
[1.5.6] Missing free memory after each restart

### ISSUE THREAD

**Issue by canedje** [athombv/homey#2250]
**Metadata**

* **Homey Version**: 1.5.6
* **Homey Log Code**:6331295BA6
* **Smartphone Platform**:
* **Smartphone App Version**:

**If related to a Homey App:**
* **Homey App Version**:
* **Homey App Log Code**:
* **Affected Devices (preferably product codes)**:

**What did you try to achieve?**

I did several restarts/ ptp with Homey

**What did you expect as result?**

The amount of free memory stay the same after each restart.

**What was the result?**

After every reset and start up free memory is missing. Without changing something within Homey
After update to 1.5.6 there was 1.32 Gb free off 1.64 GB available
After first reset there was 1.29 Gb left
Now after about 5 resets there is still 1.21 Gb left

**Any other remarks**

**Comment by bvdbos** [comment]
@canedje You know that without a log, precise version numbers etc Athom can't do anything with a bugreport?

**Comment by canedje** [comment]
The version number of Homey is mentioned in the report. What version numbers more you need?
I just send a log: nr 6331295BA6

**Comment by jeroenvollenbrock** [comment]
Could you reproduce the problem by restarting homey and then send a log about 5 minutes after the reboot?

**Comment by canedje** [comment]
I just did a reset of Homey.
Before reset there was 1,2 Gb. After reset there was 1,17 Gb left.
I did send you a LOG after about 5 minutes after restart : 77FE3A59C5

**Comment by canedje** [comment]
I also missing free memory after a while whitout restarting Homey.
I now do have 1.15 Gb left

**Comment by bvdbos** [comment]
Then probably the swap and also the insights will have grown has grown? But that's not what your issue is about is it? You specifically mention after restarts?

**Comment by canedje** [comment]
Yes. But beside that free memory is used faster then expected in normal situatioin.

**Comment by jeroenvollenbrock** [comment]
This bug should be fixed in 1.5.11, and the storage should be freed again. Could you confirm this?

**Comment by canedje** [comment]
I can not confirm.
The memory is not going down anymore. And is steady at 1.09 Gb
![knipsel](https://user-images.githubusercontent.com/23150947/39889709-e9e4ca9e-5498-11e8-9f3d-be2105db10dd.JPG)
system is still 221 MB (It was less in the beginning)

**Comment by bvdbos** [comment]
You're mixing up memory and storage. 
The OS of Homey will fill up all your memory (making every app, system and Homey as large as possible) and will then create a swap-file which means your free storage will go down (diskspace is used as memory). This issue, so I gather, is that after every reboot the used storage-space declined. You have no insight in what part of the storage grew, was it insights?

**Comment by canedje** [comment]
No I'm not mixing up.
Memory (Geheugen) is using a swapfile.
Storage (opslag) is growing after every reboot.
It is not growing anymore since a month or so. But it is also not giving memory back as in the beginning.
